plugin/vscode/extension.js

```diff
@@ -226,8 +234,7 @@ class DocumentLinter {
       this._pendingChanges.length = 0;
       this._state = DocumentLinterState.PARSER_LOADED;
       // END CRITICAL SECTION (no awaiting above)
-
-      let diags = this._parser.lint();
+      let diags = this._parser.lint(this._performanceWriter);
```

Comment:
Blocking: This looks sketch.

Reply:
Fixed in latest commit:
this._performanceWriter is deleted